assets/js/blocks/price-filter/block.tsx

```diff
@@ -28,7 +27,7 @@ import {
  */
 import usePriceConstraints from './use-price-constraints';
 import './style.scss';
-import { Attributes } from './types';
+import { PriceFilterBlockProps } from './types';
```

Comment:
Similar to your other PR, I reviewed before, what do you think about changing 
```
import { PriceFilterBlockProps } from './types';`
```
to
```
import type { PriceFilterBlockProps } from './types';
```

Reply:
Yes no problem, I'll add type here as well.